### DISCUSSION TITLE
Compatibility for Windows for ARM?

### DISCUSSION THREAD

**Discussion by PythonBlue** [apohl79/audiogridder#1164] [Ideas]
Hello again,
This is more of a long-term concern for me since my current Mac is still Intel-based, but all the same, having found a service that allows me to cloud compute with an Apple Silicon Mac, I was tempted to try out an AudioGridder server using a Parallels VM running Windows 11 for ARM. The old 64-bit version of EmulatorX VST somehow proves even more problematic in that than in the bug I reported earlier on: it won't cache it properly even when I use a version of AG that I confirmed caches it on an Intel machine. Not sure if it's the fact that it's a VST2 on an ARM-based OS (a VST3 I tested as well did cache fine), but all the same, since Windows for ARM seems to be unsupported anyway judging from the downloads available, I feel it's worth bringing up as an idea to officially support later on for the sake of other one-machine Mac users wanting to use Windows plugins, whether or not that fixes this particular use case.

**Comment by apohl79** [comment]
Added a feature request. Thx.

**Reply by PythonBlue** [reply]
Thanks, though in the meantime, I did get the VST somewhat functional in AudioGridder using JBridge and a few Windows-for-ARM-specific emulation settings. I just figure it could be useful for people who want to use Intel-based VST's on an ARM platform who don't have the technical know-how to make it fully functional.

**Reply by apohl79** [reply]
Ok, my understanding was to support AG on windows ARM natively. So AG would be able to load windows ARM plugins if it runs on ARM itself. For cross architecture functions (loading an intel plugin into an ARM host for example) I suggest to continue to use emulation bridge solutions.

**Reply by PythonBlue** [reply]
That was my intention, yes, especially since the VST I was using may be one of the lucky early VST's that does somewhat work in AudioGridder on Windows for ARM as is. Sorry for that confusion.
